Gaijinizer patch/Map066.txt

```diff
@@ -57,7 +57,7 @@ inserted into the keyholes.#Aesthetic
 　　　　　　茜の遺書
 > CONTEXT: Map066/events/7/pages/0/2/Dialogue
 > CONTEXT: Map283/events/7/pages/0/2/Dialogue
-Akane's will#MTLed
+Akane's Last Testament#Ron
```

Comment:
Above this are 3 groups of lines probably triggered by going for the Memento without getting the requirements. They'll need to be translated...